File: app/src/main/java/ru/github/pvtitov/senatapp/pojos/AuthRequest.java
package ru.github.pvtitov.senatapp.pojos;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AuthRequest {

    @SerializedName("username")
    @Expose
    private String username;
    @SerializedName("password")
    @Expose
    private String password;
    @SerializedName("rememberMe")
    @Expose
    private boolean rememberMe;

    public String getUsername() {
        return username;
    }

    public AuthRequest setUsername(String username) {
        this.username = username;
        return this;
    }

    public String getPassword() {
        return password;
    }

    public AuthRequest setPassword(String password) {
        this.password = password;
        return this;
    }

    public boolean isRememberMe() {
        return rememberMe;
    }

    public AuthRequest setRememberMe(boolean rememberMe) {
        this.rememberMe = rememberMe;
        return this;
    }

}

File: app/src/main/java/ru/github/pvtitov/senatapp/pojos/AuthRequest.kt
package ru.github.pvtitov.senatapp.pojos

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AuthRequest {

    @SerializedName("username")
    @Expose
    private var username: String = ""
    @SerializedName("password")
    @Expose
    private var password: String = ""
    @SerializedName("rememberMe")
    @Expose
    private var rememberMe: Boolean = false

    fun getUsername(): String? {
        return username
    }

    fun setUsername(username: String): AuthRequest {
        this.username = username
        return this
    }

    fun getPassword(): String? {
        return password
    }

    fun setPassword(password: String): AuthRequest {
        this.password = password
        return this
    }

    fun isRememberMe(): Boolean {
        return rememberMe
    }

    fun setRememberMe(rememberMe: Boolean): AuthRequest {
        this.rememberMe = rememberMe
        return this
    }

}
